pkgs/development/python-modules/pyezminc/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage, isPy3k, fetchFromGitHub, stdenv,
+  netcdf, hdf5, libminc, ezminc,
+  cython, numpy, scipy
+}:
+
+buildPythonPackage rec {
+  pname = "pyezminc";
+  version = "1.2";
+  name = "${pname}-${version}";
+ 
+  disabled = isPy3k;  # ??
+  # sort of works but the example programs fail due to unicode issues, etc.
+
+  src = fetchFromGitHub {
+    owner  = "BIC-MNI";
+    repo   = "pyezminc";
+    rev    = "b9fc331c5ad296348fa1ac39df8b0e26ed667d85";
+    sha256 = "1qf115d82pyw956r9jcl9xxy03smfxz7ayj1chixd1yaxsqvcigs";
+  };
+
+  nativeBuildInputs = [ cython ];
+  buildInputs = [ netcdf hdf5 libminc ezminc ];
+  propagatedBuildInputs = [ numpy scipy ];
+
+  preBuild = "export CFLAGS='-fpermissive'";
+
+  doCheck = false;  # e.g., expects test data in /opt
```

Comment:
Probably needs patching in `postPatch`: https://github.com/BIC-MNI/pyezminc/search?q=opt&unscoped_q=opt

Reply:
Is there a package that includes `share/icbm152_model_09c`?

Reply:
I think they're available as a tarball somewhere so in principle they could be included in this package or another one (e.g. BEAST, not yet Nixified)